packages/router-bridge/src/sagaRouter/module.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+
+import { connectRouter, routerMiddleware, ConnectedRouter } from 'connected-react-router';
+import { fork, takeLatest } from 'redux-saga/effects';
+
+import cmf from '@talend/react-cmf';
+import { sagaRouter, SAGA_ROUTER_HISTORY_CHANGE } from './sagaRouter';
+
+const { history }: { history: any } = require('../router');
+
+const mergeConfig = {
+	history: cmf.module.merge.getUnique,
+	sagaRouterConfig: cmf.module.merge.mergeObjects,
+	titleRouterConfig: cmf.module.merge.mergeObjects,
+	routerFunctions: cmf.module.merge.mergeObjects,
```

Comment:
This isn't used here. This is supposed to be router enter/leave functions that cmf calls when entering/leaving a route. It's not a feature of RR anymore since v4. You can remove that.